### PR TITLE
Improve README, minor changes in procedural example

### PR DESCRIPTION
**Pull request by opcode81**

### CODE REVIEW

**Review by MischaPanch — APPROVED** [review]
LGTM, feel free to merge, unless you want to continue working on this PR

**Comment by opcode81** [comment]
> LGTM, feel free to merge, unless you want to continue working on this PR

I don't have the rights to do that.

**Comment by MischaPanch** [comment]
That's weird, you have maintainer rights, which includes push. I guess it's due to the PR rules, which require the forever failing GPU test to pass - maybe maintainers are not allowed to bypass them. I'm going to remove the GPU workflow in the next PR

**Pull request merged**
